services/basic_types.proto

```diff
@@ -1592,3 +1594,15 @@ message StakingInfo {
         int64 staked_node_id = 6;
     }
 }
+
+message VirtualAddress {
+    /**
+     * The 20-byte EVM address.
```

Comment:
nit. Add explicit note to says this is the keccak-256 hash of a ECDSA_SECP256K1 primitive key

Reply:
Added.